src/ximea_ros_cam.cpp

```diff
@@ -216,7 +218,9 @@ void XimeaROSCam::initCam() {
         std::string("INVALID"));
     ROS_INFO_STREAM("calibration file: " << this->cam_calib_file_);
     this->private_nh_.param("poll_time", this->poll_time_, -1.0f);
-    ROS_INFO_STREAM("poll_time_: " << this->poll_time_);
+    ROS_INFO_STREAM("poll_time: " << this->poll_time_);
+    this->private_nh_.param("poll_time_frame", this->poll_time_frame_, 0.0f);
```

Comment:
does the 0.0f mean that is the default?

Reply:
yes, if we don't limit it, it will just default to max speed

Reply:
idk if I like that, but I'll accept it

Reply:
good stuff good stuff